src/main/resources/META-INF/plugin.xml

```diff
@@ -87,6 +87,9 @@
   ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>0.3.0</b></p>
+  <p>Stack Analysis capability is implementation to see detailed analysis of stack.</p>
```

Comment:
Shouldn't be 'is implemented ...' ?

Reply:
Ohh Sorry I was writing "Implementation of SA capabilities" then I changed it, and missed it.